flows/flows.py

```diff
@@ -185,10 +186,14 @@ def load_data(self, path: str, files_list: list, rows_amount: int = 0) -> Tuple[
         print(term.green_on_black("average_score, is_outlier = flow.outliers_extractor"
                                   "(dataframe_dict: dict, key_i: str)"))
 
-        _, _, possible_problems = detect_id_target_problem(dataframes_dict)
+        possible_ids, possible_target, possible_problems = detect_id_target_problem(dataframes_dict)
 
         _ = check_if_target_columns_are_imbalanced(dataframes_dict, possible_problems, self.kl_div_threshold)
 
+        # Check adversarial validation
+        ignore_columns = list(set(possible_ids).union(set(possible_target)))
```

Comment:
ok, created issue 

> adversarial_validation(dataframes_dict, ignore_columns)
> I would suggest adding a new list of dataframes names. Maybe the user has three 3 dataframes(datasets)[trian, validate, test] and the user interested in applying the test between validate and test, the user should be able to do this by applying the following:
> adversarial_validation(dataframes_dict, ignore_columns, datasets=["validate", "test"]) or
> adversarial_validation(dataframes_dict, ignore_columns, datasets=["train", "test"]) or
> adversarial_validation(dataframes_dict, ignore_columns, datasets=["train", "validate"])

created #43 for it

Reply:
> I would suggest keeping this (ignore_columns) a set. I discussed with Ahmad before sometimes ago and we decided to refactor all ignore_columns from lists to sets and I opened an issue for it #26. This is because it is more efficient.

ok, let's keep it in the #26 scope - no immediate solution here. For now, the only list is accessible...